content/content-tool/tool/src/webapp/vm/resources/sakai_create_uploads.vm

```diff
@@ -15,7 +15,7 @@
     #if ($itemAlertMessage)
         <div class="sak-banner-success">$itemAlertMessage</div>
     #else
```

Comment:
does the javascript logic still work properly if we remove the else?

Reply:
- [ ] @profmikegreene provide a link to the javascript method that is making this work properly.

Reply:
https://github.com/sakaiproject/sakai/blob/master/content/content-tool/tool/src/webapp/vm/resources/sakai_properties_scripts.vm#L1123 injects the message and needs to be updated to look for the new sak-banner-error class.

Reply:
Thank you for finding this, @profmikegreene! I wonder how many other cases like this we broke when we changed the banner classes. I wasn't aware that this is how some of the message banners were being populated.